src/mixxxmainwindow.cpp

```diff
@@ -745,16 +747,21 @@ void MixxxMainWindow::connectMenuBar() {
     }
 
     if (m_pCoreServices->getTrackCollectionManager()) {
-        connect(m_pMenuBar,
-                &WMainMenuBar::rescanLibrary,
-                m_pCoreServices->getTrackCollectionManager().get(),
-                &TrackCollectionManager::startLibraryScan,
-                Qt::UniqueConnection);
+        // MV: This internally disables the rescan library
```

Comment:
Is this a debug leftover?

Reply:
@mvsolves  the entire first commit is unrelated and can be removed.
When you rebase, please try to find more compact commit messages (title). If a commit needs more explanation use the message body which can also have longer lines.
IIRC we linked some resources on the wiki, but I don't find it hehe, but this looks good, too https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/

Reply:
@daschuer yep. It was from a previous branch I was working on (still learning about branches) and somehow created a branch that was on top of the other branch.